
From: "Luiz Fernando N. Capitulino" <lcapitulino@prefeitura.sp.gov.br>

Luiz has ~40 patches to his name, go he gets a banana.

Signed-off-by: Luiz Capitulino <lcapitulino@prefeitura.sp.gov.br>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/CREDITS |    8 ++++++++
 1 files changed, 8 insertions(+)

diff -puN CREDITS~credits-update CREDITS
--- 25/CREDITS~credits-update	2004-06-28 13:16:24.134834200 -0700
+++ 25-akpm/CREDITS	2004-06-28 13:16:24.140833288 -0700
@@ -3613,6 +3613,14 @@ D: MD driver
 D: EISA/sysfs subsystem
 S: France
 
+N: Luiz Fernando N. Capitulino
+E: lcapitulino@terra.com.br
+E: lcapitulino@prefeitura.sp.gov.br
+W: http://www.telecentros.sp.gov.br
+D: Little fixes and a lot of janitorial work
+S: E-GOV Telecentros SP
+S: Brazil
+
 # Don't add your name here, unless you really _are_ after Marc
 # alphabetically. Leonard used to be very proud of being the 
 # last entry, and he'll get positively pissed if he can't even
_
